src/appkit.cs

```diff
@@ -27689,4 +27861,148 @@ interface NSSharingServicePickerToolbarItemDelegate
 		[Export ("itemsForSharingServicePickerToolbarItem:")]
 		NSObject[] GetItems (NSSharingServicePickerToolbarItem pickerToolbarItem);
 	}
+
+	[Unavailable (PlatformName.MacCatalyst)]
+	[Mac (10,16)]
+	[BaseType (typeof(NSToolbarItem))]
```

Comment:
missed it earlier, same as others
`[DisableDefaultCtor]` since `itemIdentifier` can' t be `null` on the designated initializer and there's not other way to set it after it's created